types/metro-resolver/index.d.ts

```diff
@@ -1,11 +1,14 @@
-// Type definitions for metro-resolver 0.66
+// Type definitions for metro-resolver 0.72
```

Comment:
As far as I can tell, this is still valid for 0.66. I don't think we should bump this number. It would probably also be confusing since this will likely be published as 0.66.x

Reply:
Got it, I guess I misunderstood the guide, will rollback it

Reply:
done